testsuite/integration/basic/src/test/java/org/wildfly/test/integration/jmx/rbac/AbstractJmxAccessFromDeploymentWithRbacTest.java

```diff
@@ -89,7 +96,18 @@ public abstract class AbstractJmxAccessFromDeploymentWithRbacTest {
     static Archive<?> deploy(boolean securedApplication) {
         WebArchive war = ShrinkWrap.create(WebArchive.class,"jmx-access-rbac.war");
         war.addPackage(JmxResource.class.getPackage());
-        war.addAsManifestResource(AbstractJmxAccessFromDeploymentWithRbacTest.class.getResource("jboss-deployment-structure.xml"), "jboss-deployment-structure.xml");
+        war.addAsManifestResource(AbstractJmxAccessFromDeploymentWithRbacTest.class.getResource("jboss-deployment-structure.xml"), "jboss-deployment-structure.xml")
+                .addAsManifestResource(PermissionUtils.createPermissionsXmlAsset(
+                        new RuntimePermission("shutdownHooks"),
```

Comment:
@jamezp what's this one from?

Reply:
I'll remove it and look. I don't remember now :)

Reply:
@bstansberry It's the `JMXConnectorFactory.connect()`. I'll add a comment to the permission.